external/anomalib/init_venv.sh

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/env bash
+set -v
+set -x
+
+work_dir=$(realpath "$(dirname $0)")
+
+venv_dir=$1
+PYTHON_NAME=$2
+
+if [ -z "$venv_dir" ]; then
+  venv_dir=$(realpath -m ${work_dir}/venv)
+else
+  venv_dir=$(realpath -m "$venv_dir")
+fi
+
+if [[ -z $PYTHON_NAME ]]; then
+  # the default option -- note that the minimal version of
+  # python that is suitable for this repo is python3.7,
+  # whereas the default python3 may point to python3.6
+  PYTHON_NAME=python3
+fi
+
+PYTHON_VERSION=$($PYTHON_NAME --version | sed -e "s/^Python \([0-9]\.[0-9]\)\..*/\1/") || exit 1
+if [[ $PYTHON_VERSION != "3.7" && $PYTHON_VERSION != "3.8" && $PYTHON_VERSION != "3.9" ]]; then
```

Comment:
python3.7 is not supported because anomalib used `SupportsIndex`, that added in python3.8